### DISCUSSION TITLE
Migrate from next.js

### DISCUSSION THREAD

**Discussion by webdeb** [blitz-js/blitz#2054] [Q&A]
Hi folks,
has anyone tried to move an existing next.js codebase to blitz?

I am already having a quite mature nextjs codebase. With the API layer and so on. Also I am fetching data from a graphql service which is not next with apollo, basically as a wrapper of `_app.js`

Right now, I am facing some limitations for upcoming features. And I would like to partially try out the fullstack (non-api) approach of blitz. 
Is there someone who already did it, moving from next to blitz? How was the process?

**Comment by beerose** [comment]
Hey! It's been a while since this discussion started, but we recently added docs on migrating to Blitz: https://blitzjs.com/docs/migrating-to-blitz and maybe it will still be useful to you :)
